src/inflators/audio-params.ts

```diff
@@ -0,0 +1,11 @@
+import { HubsWorld } from "../app";
+import { AudioSettings } from "../components/audio-params";
+import { updateAudioSettings } from "../update-audio-settings";
+
+export function inflateAudioParams(world: HubsWorld, eid: number, params: AudioSettings) {
+  APP.audioOverrides.set(eid, params);
+  const audio = APP.audios.get(eid);
+  if (audio) {
+    updateAudioSettings(eid, audio);
+  }
```

Comment:
I don't think you need to call `updateAudioSettings` here, because reaching into `APP.audios` is analogous to reaching into `world.eid2obj`. That is -- it's outside of the scope of responsibilities of an inflator. 

I am surprised to see that there is a bitecs `AudioParams` component defined in `bit-components.js`, which is not used here.

That said, I don't think you should have the same information duplicated in multiple places, so if `APP.audioOverrides` is going to continue to exist, I we should probably not define a bitecs `AudioParams` component with all the same values. (A tag component might be necessary for the purpose of querying, but I'm not sure of even that.)

Reply:
I agree, I think this is code I moved at an early stage and never updated. It's worth noting that I added commit after the initial review that adds support for updating the audio params in media video using a `MediaLoaded` state component to set them on the actual audio entity as this inflator does not yet know about it at this stage. It's not ideal as we need to duplicate the audio params in `audioOverrides` but so far it's what I came up with.